samples/react-command-print/src/extensions/print/PrintCommandSet.ts

```diff
@@ -59,7 +59,7 @@ export default class PrintCommandSet extends BaseListViewCommandSet<IPrintComman
         dialog.itemId = event.selectedRows[0].getValueByName('ID');
         dialog.title = event.selectedRows[0].getValueByName('Title');
         dialog.show().then(()=>{
-          Dialog.alert(''); //This line prevents issues with the dialog, please do not remove it/
+          //Dialog.alert(''); //This line prevents issues with the dialog, please do not remove it/
```

Comment:
if we commented this line and confirmed the issue is not happening anymore then maybe lets remove it. What do you think?

Reply:
I have added a log instead.

```
        dialog.show().then(()=>{
          Log.info(LOG_SOURCE, 'Dialog open');
        });
```